indexer/op-legacy/predeploys.go

```diff
@@ -0,0 +1,11 @@
+package op_legacy
+
+import "github.com/ethereum/go-ethereum/common"
+
+const (
+	LegacyERC20ETH = "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000"
```

Comment:
Not the end of the world but did like having this in `op-bindings` near the other predeploy addresses

Reply:
ah i can definitely put it back in there. It was removed so I wasn't 100% sure

Reply:
moved it back